packages/cli/README.md

```diff
@@ -240,6 +240,21 @@ Run plugins, collect results and upload the report to the Code PushUp portal.
 
 Refer to the [Common Command Options](#common-command-options) for the list of available options.
 
+### `compare` command
+
+Usage:
+`code-pushup compare [options]`
```

Comment:
I always considered `[arg]` to be optional. Since this command has two required arguments, the command will always need the `--before` and `--after`. The usage should reflect that as accurately as possible, imo.

Reply:
Good point :+1: Updated.